zap-widget/src/components/SearchBox.tsx

```diff
@@ -0,0 +1,203 @@
+import { decodeInvoice } from "@getalby/lightning-tools";
+import { useEffect, useState } from "react";
+import { useClient } from "../context";
+import { cn } from "../lib/utils";
+import SimpleBoostWrapper from "./SimpleBoostWrapper";
+
+const boosts = [
+  { id: "pkg1", amount: 1 },
+  { id: "pkg2", amount: 5 },
+  { id: "pkg3", amount: 10 },
+  { id: "pkg4", amount: 25 },
+  { id: "pkg5", amount: 50 },
+  { id: "pkg6", amount: 100 },
+];
```

Comment:
You can have a const `BOOST_AMOUNTS` set to `[1,5,10,...]` and then use an index based key during mapping
```
key={`boost-${index}`}
```